Makefile

```diff
@@ -45,7 +45,7 @@ DOCKER_ENVS := \
 	-e DOCKER_USERLANDPROXY \
 	-e TESTDIRS \
 	-e TESTFLAGS \
-	-e TIMEOUT
+	-e TIMEOUT \
```

Comment:
not going to harm anything at the moment, but this line continuation isn't needed after removing the new variable